example/example8.cpp

```diff
@@ -70,8 +70,8 @@ class MyObject3 : public std::enable_shared_from_this<MyObject3> {
 };
 
 /// Make pybind aware of the ref-counted wrapper type (s)
-PYBIND11_DECLARE_HOLDER_TYPE(T, ref<T>);
```

Comment:
I propose leaving the semicolon here and removing it from the declaration of PYBIND11_DECLARE_HOLDER_TYPE.